AWS_Open_Data/README.md

```diff
@@ -149,3 +175,100 @@ Data are organized in "folders":
+  - `yyyy` is the year (4 digits), `0000` for pre-2012 products
+
+For further details on legacy datasets (`volume-products`) please refer to the GeoNet [volume products format description webpage](https://www.geonet.org.nz/data/supplementary/strong_motion_file_formats).
+
+Please use the following Digital Object Identifier https://doi.org/10.21420/X0MD-MV58 when using Strong Motion Data Products.
+
+## <a name="waveform"></a>Digital waveform data
+Raw waveform data recorded by the GeoNet sensor networks. 
+
+Data are provided in various formats depending on the application and include:
+
+- [miniseed waveform data](#wave-miniseed)
+- [Strong Motion waveform data](#wave-strong)
+
+Additional information around these datasets can be found on the [GeoNet website](https://www.geonet.org.nz/).
+
+All waveform raw datasets are available under the bucket and prefix `s3://geonet-open-data/waveforms/`.
+
+
+### <a name="wave-miniseed"></a>Miniseed waveform data
+Continuous waveform data recorded by the GeoNet sensor network (seismic, acoustic, tsunami monitoring gauges). Data are provided in miniSEED format.
+
+Sync of miniseed data from the GeoNet archive to the open data bucket is still ongoing.
+
+Additional information around this dataset can be found on the [GeoNet seismic waveform data page](https://www.geonet.org.nz/data/types/seismic_waveforms).
+
+All miniseed data are available under the bucket and prefix `s3://geonet-open-data/waveforms/miniseed/`.
+
+prexix | content
+-- | --
+`waveforms/miniseed/` | 24 hours long miniSEED data at different sampling rates
+
+Data are organized by date, and objects' prefixes are formed as:
+
+`waveforms/miniseed/[yyyy]/[yyyy].[doy]/[SITE].[NC]/[yyyy].[doy].[SITE].[NC]-[CHA].[LC].D` where:
+  - `yyyy` is year (four digits)
+  - `doy` is day of year (three digits)
+  - `SITE` is the 4 characters site code, upper case
+  - `NC` is the 2 characters network code, upper case
+  - `LC` is the 2 characters location code, upper case
+  - `CHA` is the 3 characters channel code, upper case
+
+For additional details on miniSEED data format please refer to the [SEED FDSN reference manual](https://www.fdsn.org/publications/).
+
+
+### <a name="wave-strong"></a>Strong Motion waveform data
+Triggered waveform data recorded by the GeoNet strong motion network. Data are provided in binary format.
+
+Additional information can be found on the GeoNet [Strong Motion Data Products page](https://www.geonet.org.nz/data/types/strong_motion).
+
+All strong motion triggered waveform data are available under the bucket and prefix `s3://geonet-open-data/waveforms/strong-motion/`
+
+prexix | content
+-- | --
+`waveforms/strong-motion/` | triggered event data
+
+Data are organized by equipment type and date and objects' prefixes are formed as:
+
+`waveforms/strong-motion/[type]/[yyyy]/[yyyy].[doy]/[SITE]/[yyyy].[doy].[hhmm].[ss].[SITE].[SN].[ext]`, where:
+  - `type` is the instrument type `[altus|building|cusp]`
```

Comment:
For this bit, can we not include the 'building' raw waveforms in the open-data-bucket? This is the structural array and there is still an ongoing discussion with the panel on data restrictions, although this datasets is actually accessible in the GeoNet data site already.

I'll discuss with Jono/Dan and will let you know (unless there has already been an agreement for array dataset to be included in this open-data bucket?)